examples/vdb-queries/src/main.py

```diff
@@ -85,13 +84,15 @@ async def on_start(self):
         await asyncio.sleep(5)
         await vehicle.Speed.subscribe(on_speed_update)
         (
-            await vehicle.Cabin.Seat.element_at(1, 1)
+            await vehicle.Cabin.Seat.Row(1)
+            .Pos(1)
             .Position.join(vehicle.Speed)
             .where("Vehicle.Cabin.Seat.Row1.Pos1.Position < 100")
             .subscribe(on_seat_speed_update)
         )
         (
-            await vehicle.Cabin.Seat.element_at(1, 1)
+            await vehicle.Cabin.Seat.Row(1)
+            .Pos(1)
```

Comment:
```suggestion
            await vehicle.Cabin.Seat.Row(1).Pos(1)
```

Reply:
can't have them in one line due to the linter.